raft_consensus.nim

```diff
@@ -0,0 +1,14 @@
+# nim-raft-consesnsus
```

Comment:
Let's remote the `odt` roadmap file above in favor of a markdown file with the same content.

Reply:
Done.